### PR TITLE
create roles and send permission

### PR DESCRIPTION
**Pull request by christianR15c**

### CODE REVIEW

File: .sequelizerc [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
-        "migrations-path": path.resolve('migrations')
-    };
-    
\ No newline at end of file
+module.exports = {
+  config: path.resolve('config', 'config.js'),
+  'models-path': path.resolve('models'),
+  'seeders-path': path.resolve('seeders'),
+  'migrations-path': path.resolve('migrations'),
+};
```
Comment (NyiingangoPacifique) [inline .sequelizerc]: why two sequelize configuration?

Comment (christianR15c) [reply .sequelizerc]: there aren't two conf, the codes in red are the ones I had, those in green are the updated

File: test/test.js [hunk 2]
```diff
@@ -22,34 +21,34 @@ describe('server connection', () => {
...
-        });
-    });
-  });
-});
+          res.should.have.status(404)
+          done()
+        })
+    })
+  })
+})
```
Comment (David-Neza) [inline test/test.js]: Everything works just fine as reviewed by the whole team!!

**Review by David-Neza — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

**Review by Julesbi — APPROVED** [review]

**Review by JPaul23 — APPROVED** [review]

**Review by NyiingangoPacifique — APPROVED** [review]

**Review by SimonImanigirimpuhwe — APPROVED** [review]
Good work!

**Pull request merged**